### PR TITLE
Added warning and help improvement for get_population, update line survey, add example

### PR DESCRIPTION
**Pull request by minouHub**
### Description
Provides more insight to the user when using `get_population`, see https://github.com/radis/radis/issues/508

### CODE REVIEW

File: radis/spectrum/spectrum.py [hunk 1]
```diff
@@ -2192,7 +2196,38 @@ def get_populations(self, molecule=None, isotope=None, electronic_state=None):
...
+                                                     label="pops. used to compute partition functions")
+            s.lines.query("v1l==0 & v2l==1 & v3l==0").plot("jl", "nl", ax=plt.gca(), kind="scatter", color="r",
+                                                           label="pops. of visible absorbing lines")
+            plt.legend()
+            plt.xlim((0,70))
         """
+        # Warn user on the meaning of these population
+        warn(
+            UserWarning(
+                """Populations valid for partition function calculation but sometimes NOT for spectra calculations, e.g. CO2.
```
Comment (erwanp) [inline radis/spectrum/spectrum.py:2225]: Can you also mzntio' the issue #508 here?

**Review by erwanp — APPROVED** [review]
I think Black problems are because you use a different version then the one from pre-commit.

I'll do the linting

**Comment by erwanp** [comment]
Added a full, new [Gallery Example](https://radis.readthedocs.io/en/latest/auto_examples/index.html)  showing populations plots for CO; as well as a line survey extract : 

image

**Comment by erwanp** [comment]
@minouHub you can merge when tests succeed

**Comment by minouHub** [comment]
nice additions @erwanp and for Black I'll .... see later ^^

**Comment by erwanp** [comment]
Example shows up properly : https://radis.readthedocs.io/en/latest/source/radis.spectrum.spectrum.html#radis.spectrum.spectrum.Spectrum.get_populations

**Pull request merged**